adzerk/transform.py

```diff
@@ -54,6 +56,12 @@ def to_spoc(decision):
     except (KeyError, ValueError) as e:
         logging.warning(str(e))
 
+    try:
+        spoc['is_video'] = bool(distutils.util.strtobool(custom_data['ctIsVideo'].strip()))
+    except (KeyError, ValueError):
+        # Don't set is_video if ctIsVideo is not present or not a boolean (e.g. an empty string)
```

Comment:
Can `custom_data['ctIsVideo']` be None? If so, `strip` will throw `AttributeError`.